projects/ngx-prx-styleguide/src/lib/tooltip/tooltip-content.component.ts

```diff
@@ -0,0 +1,194 @@
+import { Component, Input, AfterViewInit, ElementRef, ChangeDetectorRef } from '@angular/core';
```

Comment:
This came from the tooltip library, which didn't have any tests, and I'm not writing any. The only place this is used is on the adfree quick tag in Publish.